### PR TITLE
Add END_LAST_PAGE tag

### PR DESCRIPTION
**Pull request by LittleBigFox**
Add END_LAST_PAGE tag from a fresh cloning (without TAB but whitespace)
Can you remove the previous pull request?

### CODE REVIEW

File: html2pdf.class.php [hunk 1]
```diff
@@ -6473,6 +6473,34 @@ protected function _tag_close_G($param)
...
+         * @param  array $param
+         * @return void
+         */
+        protected function _tag_open_END_LAST_PAGE($param)
+        {
+          $height =  $this->parsingCss->ConvertToMM($param['end_height'], $this->pdf->getH() - $this->pdf->gettMargin()-$this->pdf->getbMargin());
+
+          if ($height < ($this->pdf->getH() - $this->pdf->gettMargin()-$this->pdf->getbMargin()) && $this->pdf->getY() + $height>=($this->pdf->getH() - $this->pdf->getbMargin()))
+            $this->_setNewPage();
+				}
```
Comment (spipu) [inline html2pdf.class.php]: you have again tabs ;)

**Comment by LittleBigFox** [comment]
By the hell ^^

I operate a "search and replace", i think it's better...

**Comment by spipu** [comment]
you can also modify the configuration of you IDE to do it automatically :)

**Comment by spipu** [comment]
Hi !

your pool request was good except the pb of TAB.

it will be good if i could integrate it after you modify this point :)

Best Regards,

Laurent

**Pull request merged**